app/controllers/notes_controller.rb

```diff
@@ -103,7 +103,11 @@ def create
         else
           flash[:notice] = "Research note published. Get the word out on <a href='/lists'>the discussion lists</a>!"
         end
-        redirect_to @node.path
+        if params[:redirect] && params[:redirect] == 'question'
```

Comment:
Instead of doing this here, can we do it inside the `@node.path` method, which we already use to switch between types? I think we should be able to based on whether there are tags on the note, since they'll have been added by this point.